file_util/tests/test.hpp

```diff
@@ -0,0 +1,9 @@
+#ifdef FAIL
```

Comment:
Yeah, I guess it just needs the catch include. Moved that into the test class itself.